manifest-generator/generator-services/src/main/java/io/hyscale/generator/services/builder/DefaultPortsBuilder.java

```diff
@@ -93,6 +89,21 @@
         return manifestSnippetList;
     }
 
+    public Port updatePortProtocol(Port port) {
+        port.setPort(updatePortProtocol(port.getPort()));
+        return port;
+    }
+
+    public String updatePortProtocol(String port) {
+        String[] portAndProtocol = port.split("/");
+        String protocol = DefaultPortsBuilder.ServiceProtocol.TCP.getProtocolString();
+        if (portAndProtocol.length > 1) {
+            protocol = portAndProtocol[1].equalsIgnoreCase("udp") ? ServiceProtocol.UDP.getProtocolString()
+                    : DefaultPortsBuilder.ServiceProtocol.fromString(portAndProtocol[1]).getProtocolString();
+        }
+        return portAndProtocol[0] + seperator + protocol;
+    }
+
```

Comment:
Add null check since it is a public method.
In line 101 do we need to have a separate logic for udp? Can we combine 99-103 into a conditional operator?